package.json

```diff
@@ -114,6 +117,7 @@
   },
   "peerDependencies": {
     "@dialpad/dialtone": "^7.30.0 || ^8.21.2",
+    "@dialpad/dialtone-icons": "^4.0.0",
```

Comment:
dialtone-icons is a dep and also a peer dep?

Reply:
That's a left-over when I was trying to reduce bundle size, I'll remove it

Reply:
yeah lets do this for now. I think we can maybe improve this but it'll require more research